### DISCUSSION TITLE
能补一下 checkbox 中 skipGroup 的说明吗？

### DISCUSSION THREAD

**Discussion by hellowuxin** [ant-design/ant-design#45667] [General]
### What problem does this feature solve?

找了很久如何可以不受 checkbox.group 的控制，最终在类型里找到了，但好像官方没有任何说明
这个属性还是很有用的，希望文档中可以补充

### What does the proposed API look like?

无

**Comment by afc163** [comment]
什么 skipGroup ？

**Reply by hellowuxin** [reply]
![image](https://github.com/ant-design/ant-design/assets/37543498/9e323f5d-94e1-413a-a114-5b01805a54ac)
![image](https://github.com/ant-design/ant-design/assets/37543498/4dade4c3-c916-4c51-9341-7253a3e4f0a0)

**Reply by afc163** [reply]
居然还有这个属性。。。官方没放出来是一般是没想清楚。

**Reply by hellowuxin** [reply]
但确实是我所需要的【狗头
我这里的场景是一个树状数据的勾选，用表格进行体现，在最外层包了checkbox.group后，理想状况是只控制叶子节点的勾选状态，父节点的状态由我自己计算
最后找到了skipGroup，感觉还挺符合，就是帮助部分checkbox脱离了group的控制

**Comment by zombieJ** [comment]
这个属性来自 #28576 为解决 Table 框选问题，不是对外的 API。

参考 [FAQ: 官方文档中没有提供的隐藏 API 我可以使用吗？](https://ant-design.antgroup.com/docs/react/faq-cn#%E5%AE%98%E6%96%B9%E6%96%87%E6%A1%A3%E4%B8%AD%E6%B2%A1%E6%9C%89%E6%8F%90%E4%BE%9B%E7%9A%84%E9%9A%90%E8%97%8F-api-%E6%88%91%E5%8F%AF%E4%BB%A5%E4%BD%BF%E7%94%A8%E5%90%97)，内部属性使用需要自行保证可用性。

 你的需求，自身通过 Context 封装一个 Checkbox.Group 也能实现哈~
